test/parallel/test-net-connect-buffer.js

```diff
@@ -20,68 +20,63 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
-let dataWritten = false;
-let connectHappened = false;
-
-const tcp = net.Server(function(s) {
+const tcp = net.Server(common.mustCall((s) => {
   tcp.close();
 
   console.log('tcp server connection');
```

Comment:
I would also remove the remaining `console.log()`, they don't seem to be very useful.

Reply:
Done! :)